tf2onnx/onnx_opset/tensor.py

```diff
@@ -738,19 +738,19 @@ def version_10(cls, ctx, node, **kwargs):
         #                 @int shrink_axis_mask, @int new_axis_mask)
         # T output = Slice(T input, Tind starts, Tind ends, Tind axes, Tind steps)
         # "ends" are exclusive, "axes" and "steps" are optional, their default val are [0, ...] and 1
+        input_x = node.inputs[0]
         begin = node.inputs[1]
         end = node.inputs[2]
         strides = node.inputs[3]
+        new_axis_mask = node.get_attr("new_axis_mask")
+        new_axis_mask = new_axis_mask.i if new_axis_mask is not None else 0
+
         if begin.is_const() and end.is_const() and strides.is_const() \
-                and all(val == 1 for val in strides.get_tensor_value()):
+                and all(val == 1 for val in strides.get_tensor_value()) \
+                and new_axis_mask == 0:
             cls.version_1(ctx, node, **kwargs)
             return
 
-        not_supported_attr = ["new_axis_mask"]
-        for attr_name in not_supported_attr:
-            attr = node.get_attr(attr_name)
-            if attr is not None and attr.i != 0:
-                raise ValueError("StridedSlice: attribute " + attr_name + " not supported")
         onnx_dtype = ctx.get_dtype(node.input[1])
         np_dtype = utils.ONNX_TO_NUMPY_DTYPE[onnx_dtype]
```

Comment:
np_dtype and onnx_dtype not used anywhere?

Reply:
Yes it is used around line 833.